about.html

```diff
@@ -0,0 +1,113 @@
+    <script src="https://kit.fontawesome.com/a329405765.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <div class="body-overlay hidden"></div>
+  <header>
+      <section class="user-acc">
+          <div class="social-icon">
+              <i class="fa-brands fa-twitter"></i>
+              <i class="fa-brands fa-facebook"></i>
+          </div>
+          <div class="user">
+              <nav class="nav-user">
+                  <ul>
+                      <li class="logout">Logout</li>
+                      <li class="mypage">My page</li>
+                      <li class="langue">English</li>
+                  </ul>
+              </nav>
+          </div>
+      </section>
+      <section class="main-menu">
+          <a href="./index.html"><img src="./image/logo.png" alt="logo" class="logo" width="45"></a>
+          <nav class="main-nav">
+              <i class="fa-sharp fa-solid fa-bars hamberger"></i>
+              <i class="fa-solid fa-xmark close hidden"></i>
+              <ul class="menu">
+                  <li class="about menu-i"><a href="./index.html">Home</a></li>
+                  <li class="class menu-i"><a href="#">Class</a></li>
+                  <li class="join menu-i"><a href="#">Join</a></li>
+                  <li class="teacher menu-i"><a href="#">Teacher</a></li>
+                  <li class="news menu-i"><a href="#">News</a></li>
+              </ul>
+              <a href="" class="event"> BIM Event</a>
+          </nav>
+      </section>
+    </header>
+    <main>
+        <section class="header">
+            <div class="header-overlay"></div>
+            <p class="sub-header">"Hello! Sharing world"</p>
+            <h1 class="ab-header">Bim academy training</h1>
+            <hr>
+            <p class="description-h">It feels like the world is moving at a faster pace than ever before. New building regulations, the climate crisis, net zero, ever changing sustainability and safety standards and targets – if any of these are on your mind, BIM Academy can help. Developed and run by BIM Group with over 100 years of built environment knowledge, BIM Academy regularly updates materials to reflect these changes.</p>
+            <div class="contact-info">
+                <p>Please contact us per Email for any further question about BIM Academy!</p>
+                <span class="email">bim-academy@gmail.com</span>
+            </div>
+        </section>
+        <section class="emphasic">
+            <h3>BIM Academy is the training arm of the Institution of Civil Engineers</h3>
+            <hr>
+            <p>Whether you are new or aspiring to the profession, or an experienced practitioner, we offer engineering and construction training courses that satisfy knowledge requirements across the breadth of infrastructure, as well as training on skills such as business and project management.</p>
+            <img src="./image/logo.png" alt="iso bim class">
+        </section>
```

Comment:
- [ ] Great work adjusting the size of the logo as directed by the previous reviewer. However, the block of text was not worked on. The spacing on either side of the block of text needs to be identical to that of the template chosen. To achieve this, I would advise that you increase the `margin-right` and `margin-left` for desktop screen sizes. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/211144862-62513b32-b14e-4ee1-9bf8-52cfc81072a4.png)

### Template Design

![image](https://user-images.githubusercontent.com/77106962/211144879-c54e1af8-8375-42a7-af80-3af5d24e9109.png)

#### Hint: You can use a `margin` of `0 250px` for desktop screen sizes. 💻